src/index.js

```diff
@@ -1,14 +1,35 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const container = document.querySelector('.todo-list');
 
+const list = [{
+  description: 'Walk The Dog',
+  isCompleted: true,
+  index: 0,
+},
+{
+  description: 'Buy groceries',
+  isCompleted: false,
+  index: 1,
+},
+{
+  description: 'Cook dinner',
+  isCompleted: false,
+  index: 2,
+}];
```

Comment:
- [ ] on load our list is great rendered to the DOM :clap:  But from the project requirement you need to render them in the order of their index. When I changed the index value the order doesn't change :cry:  Please fix it in order to make your project work as expected :smile: 

I think this piece of code can help you fix it :arrow_right:   **Array.sort((a, b) => a.index - b.index**

Reply:
Thank you @codecaiine for your assistance.